flow/scripts/global_place_skip_io.tcl

```diff
@@ -0,0 +1,48 @@
+if {![info exists standalone] || $standalone} {
+  # Read lef
+  read_lef $::env(TECH_LEF)
+  read_lef $::env(SC_LEF)
+  if {[info exist ::env(ADDITIONAL_LEFS)]} {
+    foreach lef $::env(ADDITIONAL_LEFS) {
+      read_lef $lef
+    }
+  }
+
+  # Read design files
```

Comment:
You need to load liberty files otherwise the report_metics fails as that information is missing.